### PR TITLE
docs: webpack debugging sections added

### PR DESCRIPTION
**Pull request by NickIliev**
Webpack debug steps for Chrome DevTools and for VSCode added in the related sections.

### CODE REVIEW

File: docs/tooling/debugging/chrome-devtools.md [hunk 1]
```diff
@@ -22,8 +24,41 @@ Table of supported features as of NativeScript version **3.4.0**
...
+To debug on Android execute the following command: 
+```Shell
+tns debug android
+```  
+To debug on iOS execute the following command: 
+```Shell
+tns debug ios
+```
+
+For security reasons,genearated the debugging agent **can't be started automatically** from the command-line. That's why NativeScript CLI **generates a URL** which is printed on the screen instead.
```
Comment (sis0k0) [inline docs/tooling/debugging/chrome-devtools.md]: genearated -> generated

File: docs/tooling/debugging/chrome-devtools.md [hunk 2]
```diff
@@ -22,8 +24,41 @@ Table of supported features as of NativeScript version **3.4.0**
...
+tns debug android
+```  
+To debug on iOS execute the following command: 
+```Shell
+tns debug ios
+```
+
+For security reasons,genearated the debugging agent **can't be started automatically** from the command-line. That's why NativeScript CLI **generates a URL** which is printed on the screen instead. 
+
+Examplgenerated URL
```
Comment (sis0k0) [inline docs/tooling/debugging/chrome-devtools.md]: Examplgenerated URL -> Example generated URL

File: docs/tooling/debugging/chrome-devtools.md [hunk 3]
```diff
@@ -22,8 +24,41 @@ Table of supported features as of NativeScript version **3.4.0**
...
+```Shell
+# NativeScript Debugger started #
+To start debugging, open the following URL in Chrome:
```
Comment (sis0k0) [inline docs/tooling/debugging/chrome-devtools.md]: I think one of the two sentences is enough.

File: docs/tooling/debugging/debugging.md [hunk 4]
```diff
@@ -40,11 +43,11 @@ You can customize the `tns debug` command using any of the following options:
 * `--emulator` - Specifies that you want to debug the app in an emulator.
 * `--timeout` - Sets the number of seconds that the NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
 * `--no-watch` - If set, changes in your code will not be livesynced.
-* `--clean` - If set, forces rebuilding the native application.
+* `--clean` - If set forces rebuilding the native application.
+* `--bundle` - Provides debugging functionality for Webpack bundled applications.
```
Comment (sis0k0) [inline docs/tooling/debugging/debugging.md]: More like - builds the app with webpack before starting the debugger or sth like that.

File: docs/tooling/visual-studio-code-extension.md [hunk 5]
```diff
@@ -28,11 +28,11 @@ Click the debugging icon ![VS Code debug panel](../img/visual-studio-code-extens
 
 ### Launch an application with the debugger
 
-Choose one of the launch configurations (e.g., `Launch on iOS`) and press the `Start Debugging` button next to the menu. This will trigger a NativeScript CLI command which will build, deploy and run your app in an iOS device or emulator and attach the Visual Studio Code debugger. If you want the execution to stop on the first JavaScript/TypeScript statement set the `stopOnEntry` flag to `true` in  your `launch.json` configuration.
+Choose one of the launch configurations (e.g., `Launch on iOS`) and press the `Start Debugging` button next to the menu. This will trigger a NativeScript CLI command which will build, deploy and run your app in an iOS device or emulator and attach the Visual Studio Code debugger. If you want the execution to stop on the first JavaScript/TypeScript statement,functionallity set the `stopOnEntry` flag to `true` in  your `launch.json` configuration.
```
Comment (sis0k0) [inline docs/tooling/visual-studio-code-extension.md]: functionallity -> functionality

File: docs/tooling/visual-studio-code-extension.md [hunk 6]
```diff
@@ -44,17 +44,46 @@ If you are writing your app in TypeScript, you have fully functional debugging s
...
+    "appRoot": "${workspaceRoot}",
+    "sourceMaps": true,
+    "watch": true,
+    "sourceMapPathOverrides": {
+        "webpack:///*": "${workspaceRoot}/app/*"
+    },
+    "tnsArgs":["--bundle"]
+},
+```
+Note that the `nativescript-dev-webpack` will autogenerate the `sourceMapPathOverrides` key.
```
Comment (sis0k0) [inline docs/tooling/visual-studio-code-extension.md]: The VSCode extension generates that key.

**Review by sis0k0 — CHANGES_REQUESTED** [review]

**Review by tsonevn — APPROVED** [review]

**Comment by tsonevn** [comment]
test

**Pull request merged**